src/components/FiltersContainer/index.js

```diff
@@ -1,33 +1,67 @@
-import { Divider, Row, Select, Typography } from "antd";
+import { Col, DatePicker, Divider, Row, Select, Typography } from "antd";
+import moment from "moment";
 import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  setSelectedFilter,
+  filtredFacts,
+  filtredByMonth,
+} from "../../redux/actions";
 
 const { Title } = Typography;
 const { Option } = Select;
 
 const FiltersContainer = () => {
+  const state = useSelector((state) => state);
+  const { filters, time } = state;
+  const dispatch = useDispatch();
+
+  const handleChangeSelect = (type, option) => {
+    const data = { name: option.children, value: option.value };
+    dispatch(setSelectedFilter(type, data));
+    dispatch(filtredFacts());
+  };
+
+  const getDisabledDate = (current) => {
```

Comment:
You can move it out of the body of the component, since it does not use components state, but creates an unnecessary re-declaration of functions on each rendering